### ISSUE TITLE
Bible Reference links are not working in Words

### ISSUE THREAD

**Issue by elsylambert** [unfoldingWord-dev/ts-desktop#1275]
#v12.1 Build 143

Links in Words for Bible reference and Open Bible stories are clickable but does not open any window/any results on clicking.
![image.png](https://images.zenhubusercontent.com/5d02efb2f425650e637abe10/c85a336b-408c-4834-9b27-ebab6ab96d26)

**Comment by jag3773** [comment]
@neutrinog I seem to remember that this is a known bug, is that true?

**Comment by neutrinog** [comment]
yes. those never worked. Right now there's no practical way to link to a different project without re-writing the entire program. Lessons learned...
